requirements_dev.txt

```diff
@@ -1,20 +1,19 @@
 # # Packaging controls
-setuptools>=61.2.0
-versioneer>=0.22
-packaging>=21.3
+setuptools>=64.0.0
```

Comment:
I think you can get away with not including this, as if you build the package locally this is installed using the build requirements in pyproject.toml
```suggestion
```